ccpage146.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Write a function called city_country() that takes in the name of a city and its country.
+# The function should return a string formatted like this: "Santiago, Chile"
+# Call your function with at least three city-country pairs, and print the value that’s returned.
+Option 1
+def city_country (city, country):
+    """List a city and its country"""
+    print(f"{city.title()}, {country.title()}")
+
+city_country('dublin', 'ireland')
+city_country('boston', 'united states')
+city_country('rome', 'italy')
+
+Option 2: 
+def city_country(city, country):
+    """Locate city and its country"""
+    location = city + ', ' + country
+    return location.title()
+
+city_country_location = city_country('rome', 'italy')
+print(city_country_location)
+
+city_country_location = city_country('boston', 'united states')
+print(city_country_location)
+
+city_country_location = city_country('dublin', 'ireland')
+print(city_country_location)
+
+
+# Write a function called make_album() that builds a dictionary describing a music album. 
+# The function should take in an artist name and an album title, and it should return a dictionary containing these two pieces of information.
+# Use the function to make three dictionaries representing different albums.
+Print each return value to show that the dictionaries are storing the album information correctly.
+def make_album(name, title):
+    """Describe one album"""
+    album_1 = {'first_name': name, 'album_title': title}
```

Comment:
This problem is asking you to create a function that builds _one_ dictionary and returns it. Then you can call the function three separate times and save the new albums to three separate variables.